src/style.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: "avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.big_searchbar {
+  margin: 20px auto;
+}
+
+.search {
+  width: 100%;
+  padding: 23px 62px;
+  box-sizing: border-box;
```

Comment:
Move it to `*` selector, box-sizing should be the same for all elements
```
  * {
    box-sizing: border-box;
  }
```

Reply:
All changes has been made